js_modules/dagit/packages/app/src/NUX/CommunityNux.tsx

```diff
@@ -28,12 +40,128 @@ const CommunityNuxImpl: React.FC<{dismiss: () => void}> = ({dismiss}) => {
   React.useEffect(() => {
     const timeout = setTimeout(() => {
       setShouldShowNux(true);
-    }, TIMEOUT); // wait 5 seconds before showing Nux
+    }, TIMEOUT);
     return () => {
       clearTimeout(timeout);
     };
   }, []);
 
+  const [iframeData, setIframeData] = React.useState<{email: string; newsletter: boolean} | null>(
+    null,
+  );
+  const submit = (email: string, newsletter: boolean) => {
+    setIframeData({email, newsletter});
+  };
+
+  return (
+    <Dialog isOpen={shouldShowNux} style={{width: '680px'}}>
```

Comment:
Can the width be a little smaller for the recaptcha iframe, or could the recaptcha checkbox be a little more centered? There's a decent amount of whitespace to the right of it right now.

Reply:
https://github.com/dagster-io/dagster-website/pull/391/files